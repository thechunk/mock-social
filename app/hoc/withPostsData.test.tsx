import * as React from 'react';
import {shallow} from 'enzyme';

describe('withPostsData', () => {
    it('renders correctly', () => {
        expect(true).toEqual(false);
    });
});