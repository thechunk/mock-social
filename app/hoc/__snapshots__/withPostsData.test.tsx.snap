// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPostsData wrapper renders correctly 1`] = `
<Component
  id={1}
  loading={true}
  posts={Array []}
/>
`;
